src/mainboard/clevo/tgl-u/ramstage.c

```diff
@@ -12,6 +12,7 @@
 #include <soc/ramstage.h>
 #include <variant/gpio.h>
 #include <variant/ramstage.h>
+#include <device/device.h>
```

Comment:
This include is already present on line 3

Reply:
fixed in 31f28991bb1c1c1425e8a81e98819326d217a534